.travis.yml

```diff
@@ -14,8 +14,7 @@ install: echo "skip bundle install"
 
 branches:
   only:
-    - master
-    - kitchen_refactor
+    - /.*/
```

Comment:
If you open a WIP PR Travis will always test the PR. That way we avoid double testing on PRs with local branches, which takes forever to run with all these integration tests.